src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CodeTypeReference.cs

```diff
@@ -0,0 +1,363 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Runtime.Serialization
```

Comment:
You should consider potentially sharing the source file and just conditioning the namespace or even the type visibility.